johnzon-core/src/main/java/org/apache/johnzon/core/JsonNumberImpl.java

```diff
@@ -117,4 +119,17 @@ private void checkFractionalPart() {
             throw new ArithmeticException("Not an int/long, use other value readers");
         }
     }
+
+    private void checkBigIntegerScale() {
```

Comment:
guess we need a toggle in the `Map` + `JsonbConfig` + `MapperBuilder` to disable it for backward compat - it stays legal?